README.md

```diff
@@ -50,6 +46,42 @@ Before running `linguist`, make sure:
 
   If authentication issues persist, rerun this command again.
 
+## Usage
+
+```sh
+linguist --ssh-key <SSH_KEY_PATH> [--ui-path <UI_PATH>] [--frontary-path \ <FRONTARY_PATH>]
+```
+
+### Arguments
+
+<!-- markdownlint-disable -->
+| Argument                          | Description                                                          | Required |
+|-----------------------------------|----------------------------------------------------------------------|----------|
+| `--ssh-key <SSH_KEY_PATH>`                  | Path to your SSH private key file used for GitHub operations         | Yes      |
```

Comment:
Could you please adjust whitespace so that the vertical `|` lines in Markdown table line up neatly, ensuring consistent column widths?